ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -132,6 +148,36 @@ class SqlLedgerSpec
       }
     }
 
+    "refuse to create a new ledger when there is already one with a different participant id" in {
```

Comment:
I'm a bit worried about the naming that's emerging: it seems to conflate ledgers and participants. This pattern emerges throughout the PR, I believe as part of adapting new code to existing code. Could it make sense to be more explicit about what pertains to a participant and what to the ledger that backs it?

Reply:
I'm not sure I understand. Could you elaborate please?

Reply:
As mentioned offline, I'm not sure the `participantId` should be tied to an entity called "ledger". Should we be more explicit about the fact that we are talking about a participant here? It's indeed the "gateway" to a ledger, but it's not a user-facing component (in the sense that a user would not directly use `SqlLedger` in its own code, but rather go through the Ledger API), so I see less chances to be misunderstood by referring to participants explicitly in the code.